starknet/src/types/update_settings_calldata.cairo

```diff
@@ -17,6 +20,7 @@ struct UpdateSettingsCalldata {
     voting_strategies_to_remove: Array<u8>,
 }
 
+// TODO: use `Default` trait
```

Comment:
can this be done yet?

Reply:
I tried to but wasn't working. Will try in another PR